packages/angular/projects/angular-sdk/package.json

```diff
@@ -8,8 +8,8 @@
     "prepack": "shx cp ./../../../../LICENSE ./LICENSE"
   },
   "peerDependencies": {
-    "@angular/common": "^16.2.12 || ^17.3.0",
-    "@angular/core": "^16.2.12 || ^17.3.0",
+    "@angular/common": "^16.2.12 || ^17.3.0 || ^18.2.9",
```

Comment:
Could this be `^18.0.0`?

Reply:
If I am not mistaken, this should not matter right?
Or is it the minimum version still with this?

Reply:
Yes, just found out too and committed a change :D